### PR TITLE
76_张庆杰_第一课作业

### PR DESCRIPTION
**Pull request by best-trip**

### CODE REVIEW

**Comment by best-trip** [comment]
@Smillence

**Comment by best-trip** [comment]
@ruoshuigao

File: Lesson-1/assignment/yours.sol [hunk 1]
```diff
@@ -1 +1,57 @@
...
+    }
+    
+    // 2 设置员工地址, 填入需要加双引号
+    function setAddress(address employeeAddress){
+        salaryAddress = employeeAddress;
+    }
+    
+    // 3 设置员工薪酬
+    function setAmount(uint employeeAmount){
+        salaryAmount = 1000000000000000000 * employeeAmount;
```
Comment (ruoshuigao) [inline Lesson-1/assignment/yours.sol]: `1000000000000000000 wei == 1 ether` 这里用`salaryAmount =  employeeAmount * 1 ether` 会更简洁 @jasoncheetah

**Comment by best-trip** [comment]
@ruoshuigao Ths.

**Pull request merged**